packages/jaeger-ui/src/components/TraceDiff/TraceDiffHeader/CohortTable.test.js

```diff
@@ -42,190 +37,151 @@ describe('CohortTable', () => {
       id: 'trace-id-2',
     },
   ];
-  const selectTrace = jest.fn();
-  const props = {
-    cohort,
-    current: cohort[0].id,
-    selection: {
-      [cohort[0].id]: {
-        label: 'selected index 0',
-      },
-    },
-    selectTrace,
-  };
-
+  let selectTrace;
+  let props;
   let formatDurationSpy;
-  let wrapper;
-
-  /**
-   * Creates a new wrapper with default props and specified props. It is necessary to create a new wrapper
-   * when props change because enzyme does not support wrapper.setProps for classes that render an array of
-   * elements.
-   *
-   * @param {Object} [specifiedProps={}] - Props to set that are different from props defined above.
-   * @returns {Object} - New wrapper.
-   */
-  function updateWrapper(specifiedProps = {}) {
-    wrapper = shallow(<CohortTable {...props} {...specifiedProps} />);
-  }
-
-  function getRowRenderer(dataIndex) {
-    return wrapper.find(Column).find(`[data-testid="${dataIndex}"]`).prop('render');
-  }
 
   beforeAll(() => {
     formatDurationSpy = jest.spyOn(dateUtils, 'formatDuration');
   });
 
   beforeEach(() => {
-    selectTrace.mockReset();
+    selectTrace = jest.fn();
     formatDurationSpy.mockReset();
-    updateWrapper();
+    props = {
+      cohort,
+      current: cohort[0].id,
+      selection: {
+        [cohort[0].id]: {
+          label: 'selected index 0',
+        },
+      },
+      selectTrace,
+    };
   });
 
   it('renders as expected', () => {
-    expect(wrapper).toMatchSnapshot();
+    render(<CohortTable {...props} />);
+    expect(screen.getByText('Service & Operation')).toBeInTheDocument();
+    expect(screen.getByText('Date')).toBeInTheDocument();
+    expect(screen.getByText('Duration')).toBeInTheDocument();
+    expect(screen.getByText('Spans')).toBeInTheDocument();
   });
 
   describe('row selection', () => {
-    let rowSelection;
-
-    function updateRowSelection() {
-      rowSelection = wrapper.find(Table).prop('rowSelection');
-    }
-
-    beforeEach(() => {
-      updateRowSelection();
-    });
-
     it('defaults selectedRowKeys to empty array', () => {
-      updateWrapper({ current: undefined });
-      updateRowSelection();
-      expect(rowSelection.selectedRowKeys).toEqual([]);
+      render(<CohortTable {...props} current={undefined} />);
+      expect(screen.queryByRole('radio', { checked: true })).not.toBeInTheDocument();
```

Comment:
why does this need `checked: true`? Since you didn't simulate any clicks this condition is always true.